cluster/gce/util.sh

```diff
@@ -1303,7 +1303,17 @@ function test-teardown {
     --project "${PROJECT}" \
     --quiet \
     "${NODE_TAG}-${INSTANCE_PREFIX}-nodeports" || true
-  "${KUBE_ROOT}/cluster/kube-down.sh"
+  if [[ ${MULTIZONE:-} == "true" ]]; then
+      local zones=( $E2E_ZONES )
```

Comment:
minor nit: gonna go back to my original style rec of s/`$E2E_ZONES`/`${E2E_ZONES}`/, so here
```
local zones=( ${E2E_ZONES} )
```